_includes/live-demos/comments-callback/style.css

```diff
@@ -0,0 +1,35 @@
+/*
+ * The following CSS is for styling the user drop-down menus
+ * above the editor and are not required for the Comments plugin.
+ */
+.box {
```

Comment:
Does this work on IE11, as it's flex support can be hit and miss?

Reply:
After discussion, moving the code to es6. given the rest of the demo wont work on IE, not too worried about this. not sure what will happen, but it may just wrap instead of flex, which is also fine